libs/ledgerjs/packages/hw-transport-http/src/VaultTransport.ts

```diff
@@ -0,0 +1,48 @@
+import { log } from "@ledgerhq/logs";
+import WebSocketTransport from "./WebSocketTransport";
+
+type VaultData = {
+  token: string;
+  workspace: string;
+};
+
+export default class VaultTransport extends WebSocketTransport {
```

Comment:
it would be better to make `VaultTransport` a dedicated package because it is not the scope of hw-transport-http i think.

an alternative idea is to add feature directly on the WebSocketTransport class, but it should not be breaking. it might be good to have an independent package that you can maintain independently (and that depends on hw-transport-http)

Reply:
I think you are right, vault transport should be a dedicated package!
I will work on that